.github/workflows/build-images-action.yml

```diff
@@ -0,0 +1,22 @@
+name: build-images-action
+on: 
+  push:
+    branches: [ main ]
+jobs:
```

Comment:
Please set appropriate `permissions` for the workflow top-level or under `jobs`.

Reply:
Ah, yes. I'll add it.

Reply:
Also drop all the permissions on the top-level: `permissions: {}`